modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/workflow/generators/SW360UpdaterImpl.java

```diff
@@ -49,19 +53,23 @@ public SW360UpdaterImpl(SW360MetaDataUpdater sw360MetaDataUpdater,
                         .map(SW360License::getShortName)
                         .collect(Collectors.toSet());
             }
-
-            final SW360Release sw360ReleaseFromArtifact = SW360ReleaseAdapterUtils.convertToRelease(artifact);
-            sw360ReleaseFromArtifact.setMainLicenseIds(licenseIds);
-            SW360Release sw360ReleaseFinal = sw360MetaDataUpdater.getOrCreateRelease(sw360ReleaseFromArtifact);
-            if (sw360MetaDataUpdater.isUploadSources()
-                    && sw360ReleaseFinal.get_Links().getSelf() != null
-                    && !sw360ReleaseFinal.get_Links().getSelf().getHref().isEmpty()) {
-                Map<Path, SW360AttachmentType> attachments = SW360AttachmentAdapterUtils.getAttachmentsFromArtifact(artifact);
-                if (!attachments.isEmpty()) {
-                    sw360ReleaseFinal = sw360MetaDataUpdater.uploadAttachments(sw360ReleaseFinal, attachments);
+            try {
+                final SW360Release sw360ReleaseFromArtifact = SW360ReleaseAdapterUtils.convertToRelease(artifact);
+                sw360ReleaseFromArtifact.setMainLicenseIds(licenseIds);
+                SW360Release sw360ReleaseFinal = sw360MetaDataUpdater.getOrCreateRelease(sw360ReleaseFromArtifact);
+                if (sw360MetaDataUpdater.isUploadSources()
+                        && sw360ReleaseFinal.get_Links().getSelf() != null
+                        && !sw360ReleaseFinal.get_Links().getSelf().getHref().isEmpty()) {
+                    Map<Path, SW360AttachmentType> attachments = SW360AttachmentAdapterUtils.getAttachmentsFromArtifact(artifact);
+                    if (!attachments.isEmpty()) {
+                        sw360ReleaseFinal = sw360MetaDataUpdater.uploadAttachments(sw360ReleaseFinal, attachments);
+                    }
                 }
+                releases.add(sw360ReleaseFinal);
+            } catch(Exception e) {
```

Comment:
Never catch a generic Exception, you also catch all RuntimeExceptions and the like, which should not be caught.